scripts/utility-wfrp4e.js

```diff
@@ -881,6 +881,44 @@ class WFRP_Utility
       MarketWfrp4e.generatePayCard(payString);
   }
 
+   /**
+   * Convert's a weapons length to an integer
+   * 
+   * @param {String} weaponLength the weapon's length
+   */
+  static evalWeaponLength(weaponLength)
```

Comment:
Can you put this instead in a key value pair in the WFRP4E object?

```
WFRP4E.reachNums = {
'personal' : 1,
//etc
}
```

Reply:
Oh that is a better way to do that.